### PR TITLE
Some code style optimizations and clean tainted_functions.h

### PR DESCRIPTION
**Pull request by ampresent**

### CODE REVIEW

File: boundary/analyze.py [hunk 1]
```diff
@@ -206,7 +206,7 @@ def lookup_if_global(name_and_file):
     func_class['sched_outsider'] = (func_class['mod_fns'] - func_class['insider'] - func_class['border']) | func_class['fn_ptr_optimized']
     func_class['optimized_out'] = func_class['sched_outsider'] - func_class['in_vmlinux']
     func_class['public_user'] = func_class['fn'] - func_class['insider'] - func_class['border']
-    func_class['tainted'] = (func_class['border'] | func_class['insider']) & func_class['in_vmlinux']
+    func_class['tainted'] = (func_class['border'] | func_class['insider']) & func_class['in_vmlinux'] - func_class['init']
```
Comment (chenshanpei) [inline boundary/analyze.py:209]: Nice catch. But removing func_class['init'] directly from func_class['insider'] is looks better for me. Then we don't need to treat func_class['init'] as a special case.

File: src/Makefile [hunk 2]
```diff
@@ -39,9 +39,10 @@ $(obj)/%.o: $(src)/%.c GET_STACK_SIZE FORCE
 	$(call if_changed_rule,cc_o_c)
 	$(OBJCOPY) $(OBJCOPYFLAGS) $(OBJCOPYFLAGS_$(@F)) $@
 
-scheduler-objs := $(objs-y) main.o tainted.o sched_rebuild.o
+scheduler-objs := $(objs-y) main.o bypass_test.o tainted.o sched_rebuild.o
```
Comment (chenshanpei) [inline src/Makefile:42]: Don't put test code in src !!! If you want to such kind of test, please try other ways.

File: src/main.c [hunk 3]
```diff
@@ -77,7 +77,8 @@ struct tainted_function {
 	},
 
 struct tainted_function tainted_functions[] = {
-	#include "tainted_functions.h"
+	#include "tainted_functions.h",
+	NULL
```
[the remark this answers is not in the document]
Comment (ampresent) [reply src/main.c]: How about this style? Is it comfortable for you?

```
struct tainted_function tainted_functions[] = {
	#include "tainted_functions.h",
	{.name = NULL}
}
```

File: src/Makefile (hunk 2 above)
Comment (ampresent) [reply src/Makefile]: OK I'll drop this patch

**Pull request merged**